cli/build.rs

```diff
@@ -21,8 +24,8 @@ fn extract_git_hash() -> Result<()> {
     vergen(config).map_err(|err| eyre!(Box::new(err)))
 }
 
-fn check_permission_validators() -> Result<()> {
-    iroha_wasm_builder::Builder::new(PERMISSION_VALIDATORS_PATH)
+fn check_default_validator() -> Result<()> {
```

Comment:
having thought about it, it shouldn't be called `permission_validator` because it's more broad than that. `runtime_validator` or `verifier` is more appropriate